faucet-server/Sources/App/Services/Wallet/Wallet.swift

```diff
@@ -17,15 +17,21 @@ public class Wallet {
     var deps: [OutPoint] {
         return [systemScriptOutPoint]
     }
-    var publicKey: H256 {
+    lazy var publicKey: H256 = {
         return "0x" + Utils.privateToPublic(privateKey)
-    }
-    public var lock: Script {
-        return Script.verifyScript(for: publicKey, codeHash: systemScriptCellHash)
-    }
-    var lockHash: H256 {
+    }()
+    lazy var address: String = {
+        return AddressGenerator(network: .testnet).address(for: publicKey)
+    }()
+    lazy var publicKeyHash: String = {
+        return "0x" + AddressGenerator(network: .testnet).publicKeyHash(for: address)!
+    }()
```

Comment:
Are these computing properties (`publicKey`, `address` and `publicKeyHash`) really very slow to access that they have to be lazy ones?

Reply:
It only takes about 3ms in my device, should not need to be  lazy ones 
but I don't think they need to be executed repeatedly.

Reply:
Ok